docker-entrypoint.sh

```diff
@@ -137,21 +177,24 @@ function _django_fixtures {
 
 
 function _runserver() {
-    echo "RUNSERVER_OPTS is ${RUNSERVER_OPTS}"
+    : "${RUNSERVER_OPTS=${RUNSERVER_CMD} 0.0.0.0:${RUNSERVERPORT} --settings=${DJANGO_SETTINGS_MODULE}}"
```

Comment:
Removing duplicate declarations of the same var can probably count as linting - this snippet doesn't really show that though. FWIW it wasn't identified by the linter as such.